src/symbols.js

```diff
@@ -716,36 +716,76 @@ defineSymbol(text, main, spacing, "\u00a0", " ");
 defineSymbol(text, main, spacing, "\u00a0", "~");
 
 // There are lots of symbols which are the same, so we add them in afterwards.
+let i = 0;
+let ch = "";
 
 // All of these are textords in math mode
 const mathTextSymbols = "0123456789/@.\"";
-for (let i = 0; i < mathTextSymbols.length; i++) {
-    const ch = mathTextSymbols.charAt(i);
+for (i = 0; i < mathTextSymbols.length; i++) {
+    ch = mathTextSymbols.charAt(i);
     defineSymbol(math, main, textord, ch, ch);
 }
 
 // All of these are textords in text mode
 const textSymbols = "0123456789!@*()-=+[]<>|\";:?/.,";
-for (let i = 0; i < textSymbols.length; i++) {
-    const ch = textSymbols.charAt(i);
+for (i = 0; i < textSymbols.length; i++) {
+    ch = textSymbols.charAt(i);
     defineSymbol(text, main, textord, ch, ch);
 }
 
 // All of these are textords in text mode, and mathords in math mode
-const letters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ";
-for (let i = 0; i < letters.length; i++) {
-    const ch = letters.charAt(i);
+const letters = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
+for (i = 0; i < letters.length; i++) {
+    ch = letters.charAt(i);
     defineSymbol(math, main, mathord, ch, ch);
     defineSymbol(text, main, textord, ch, ch);
 }
 
+// The next loop loads wide (surrogate pair) characters.
+// We support some letters in the Unicode range U+1D400 to U+1D7FF,
+// Mathematical Alphanumeric Symbols.
+// Some editors do not deal well with surrogate pairs. So don't write the
+// string into this file. Instead, create the string from the surrogate pair.
+let wideChar = "";
+for (i = 0; i < letters.length; i++) {
+    ch = letters.charAt(i);
+
+    // The hex numbers in the next line are a surrogate pair.
+    // 0xD835 is the high surrogate for all letters in the range we support.
+    // 0xDC00 is the low surrogate for bold A.
+    wideChar = String.fromCharCode(0xD835, 0xDC00 + i);  // A-Z a-z bold
+    defineSymbol(math, main, mathord, ch, wideChar);
+
+    // We omit italic. Math mode A-Za-z are italic by default.
+
+    wideChar = String.fromCharCode(0xD835, 0xDC68 + i);  // A-Z a-z bold italic
+    defineSymbol(math, main, mathord, ch, wideChar);
+
+    wideChar = String.fromCharCode(0xD835, 0xDD04 + i);  // A-Z a-z Fractur
+    defineSymbol(math, main, mathord, ch, wideChar);
+
+    if (i < 26) {
+        // KaTeX fonts have only capital letters for blackboard bold and script.
+        wideChar = String.fromCharCode(0xD835, 0xDD38 + i); // A-Z double struck
+        defineSymbol(math, main, mathord, ch, wideChar);
+
+        wideChar = String.fromCharCode(0xD835, 0xDC9C + i); // A-Z script
+        defineSymbol(math, main, mathord, ch, wideChar);
+    }
+
+    // TO DO: Add bold script when it is supported by a KaTeX font.
```

Comment:
Could you use `TODO`(without the space), as some IDEs track them and one may do `git grep TODO`?

Reply:
Will do.